test/e2e/network_test.go

```diff
@@ -452,11 +452,6 @@ func (tc *testContext) getPodIP(selector metav1.LabelSelector) (string, error) {
 
 // getWindowsServerContainerImage gets the appropriate WindowsServer image based on VXLAN port
```

Comment:
The function's description should be updated given the `VXLAN port` is no longer the driving condition for the selection of the container image.

Reply:
Fixed.